### PR TITLE
Removed support/migration locale folders

### PR DESCRIPTION
**Pull request by Delphine**

### CODE REVIEW

**Comment by Delphine** [comment]
Actually, if the component has been entirely removed, I assume I should delete the all the folders in it.
Waiting for Monday just in case.

**Comment by flodolo** [comment]
This is OK. But when we merge this, the next day automation will try to add back the strings until they're actually removed from `firefox-android`.

So, we need to merge this, then wait for the strings to be removed also in `firefox-android` before merging any PR coming in from a-c.

At this point, I strongly suggest to wait for confirmation on the other component (awesomebar) and remove them together if necessary.

**Review by flodolo — APPROVED** [review]

**Comment by flodolo** [comment]
As mentioned, look out for the PR for android-components in ~5 hours (it will try to add back the 2 en-US files)

**Pull request merged**
